SUPPORT.md

```diff
@@ -0,0 +1,28 @@
+# Getting help with Pony
+
+Looking for help with Pony? Awesome. We are here to help. Before you open an issue against this repo, we ask you read the following to better direct you to the proper forum.
+
+## Should I open an issue?
+
+We ask you to refrain from opening GitHub issues for non-actionable items. We use GitHub issues to track bugs and other problems that can be resolved. If you are not sure how to code a particular problem, how to decipher an error message, wondering about a benchmark, or anything of that sort, please use the [mailing list][mailing list] or [IRC][IRC].
+
+## How to get help
+
+You have a couple of different options on how you can get help from the Pony community. If you are looking for an answer "right now," we suggest you give our IRC channel a try. It's #ponylang on Freenode. If you ask a question, be sure to hang around until you get an answer. If you don't get one, or IRC isn't your thing, we have a friendly mailing list you can try. Whatever your question is, it isn't dumb, and we won't get annoyed.
```

Comment:
Maybe `#ponylang` should be in backticks?

Reply:
Updated.